services/bls_aggregation/blsagg.go

```diff
@@ -134,17 +135,24 @@ type BlsAggregatorService struct {
 	taskChansMutex     sync.RWMutex
 	avsRegistryService avsregistry.AvsRegistryService
 	logger             logging.Logger
+
+	// taskResponseMap is a map of taskResponseDigest to taskResponse
+	taskResponseMap map[types.TaskResponseDigest]types.TaskResponse
```

Comment:
I'd prefer is this was local to each task's goroutine like the other data structs, just for consistency (also no chance of having concurrency bugs if for eg there's a taskresponsedigest from two different tasks that have the same digest, however unlikely)

Reply:
2177792fa8f14b23cead9ea6a159cfca2b3c0776